src/devices/oregon_scientific.c

```diff
@@ -352,6 +353,23 @@ static int oregon_scientific_v2_1_parser(bitbuffer_t *bitbuffer) {
         // RF Clock message ??
       }
       return 1;
+    } else if (sensor_id  == ID_THN129) {
+      if ((validate_os_v2_message(msg, 137, num_valid_v2_bits, 12) == 0)) {
+        float temp_c = get_os_temperature(msg, sensor_id);
+        data = data_make(
+            "time",          "",            DATA_STRING, time_str,
+            "brand",         "",            DATA_STRING, "OS",
+            "model",         "",            DATA_STRING, "THN129",
+            "id",            "House Code",  DATA_INT,    get_os_rollingcode(msg, sensor_id),
+            "channel",       "Channel",     DATA_INT,    get_os_channel(msg, sensor_id), // 1 to 5
+            "battery",       "Battery",     DATA_STRING, get_os_battery(msg, sensor_id) ? "LOW" : "OK",
+            "temperature_C",  "Celcius",    DATA_FORMAT, "%.02f C", DATA_DOUBLE, temp_c,
+            "temperature_F",  "Fahrenheit", DATA_FORMAT, "%.02f F", DATA_DOUBLE, ((temp_c*9)/5)+32,
```

Comment:
no sure to understand what you mean, the sensor transmits, chan & bat status (in message begin after Id) then temp in degree_C. Which os these lines should I remove?